mage_ai/data_preparation/templates/data_loaders/bigquery.py

```diff
@@ -0,0 +1,22 @@
+from mage_ai.data_loader.bigquery import BigQuery
+from pandas import DataFrame
+
+
+def load_data_from_big_query() -> DataFrame:
+    """
+    Template code for loading data from Google BigQuery.
+
+    This code template assumes that the `GOOGLE_APPLICATION_CREDENTIALS` environment
+    variable is not set.
+    - If this environment variable set, then config can be left empty.
```

Comment:
I switched the constructor so that an empty config can be accepted. Now an error should be raised only if the keyword argument is not accepted by the Google BigQuery client and is neither of `path_to_credentials` and `credentials_mapping`